src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -845,22 +846,29 @@ void FancyZones::UpdateDragState(HWND window, require_write_lock) noexcept
         m_dragEnabled = !(shift | mouse);
     }
 
-    const bool windowElevated = IsProcessOfWindowElevated(window);
-    static const bool meElevated = is_process_elevated();
-    static bool warning_shown = false;
-    if (windowElevated && !meElevated)
+    // We should move this logic into the runner once we monitor all elevated processes which are being launched
+    // and notify a user about them, since other modules might care about them as well
+    static std::time_t warning_shown_at = 0;
+    constexpr int64_t warning_cooldown_interval_minutes = 60;
+    const auto now = timeutil::now();
+    const bool warning_ready = timeutil::diff::in_minutes(now, warning_shown_at) > warning_cooldown_interval_minutes;

```

Comment:
This will actually take 61 minutes :)

Reply:
I guess we have to fire yuyoyuppe.

Reply:
We should create a separate task for it, since it's not a one-minute issue.